packages/evm/src/opcodes/codes.ts

```diff
@@ -372,6 +380,19 @@ const eipOpcodes: { eip: number; opcodes: OpcodeEntry }[] = [
       0xee: asyncAndDynamicGasOp('RETURNCONTRACT'),
     },
   },
+  {
+    eip: 6690,
+    opcodes: {
+      // control & i/o
+      0xc0: asyncAndDynamicGasOp('SETMODX'),
+      0xc1: asyncAndDynamicGasOp('LOADX'),
+      0xc2: asyncAndDynamicGasOp('STOREX'),
+      // arithmetic
+      0xc3: asyncAndDynamicGasOp('ADDMODX'),
+      0xc4: asyncAndDynamicGasOp('SUBMODX'),
+      0xc5: asyncAndDynamicGasOp('MULMODX'),
```

Comment:
Note: for dynamic gas ops, a gas method should be provided in `gas.ts`, otherwise the Interpreter will crash (it expects an entry there if the opcode is marked as dynamic)